core/rawdb/database_ext.go

```diff
@@ -0,0 +1,73 @@
+// (c) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package rawdb
+
+import (
+	"bytes"
+
+	"github.com/ava-labs/libevm/common"
+	ethrawdb "github.com/ava-labs/libevm/core/rawdb"
+	"github.com/ava-labs/libevm/ethdb"
+)
+
+// InspectDatabase traverses the entire database and checks the size
+// of all different categories of data.
+func InspectDatabase(db ethdb.Database, keyPrefix, keyStart []byte) error {
+	var (
```

Comment:
Isn't this going to conflict with #850? Perhaps just add that code here?

Reply:
That's fine, I'll take this into account and handle it accordingly depending which gets merged first 😉